### ISSUE TITLE
CosmosClient to use connectionstring

### ISSUE THREAD

**Issue by mindlessroman** [Azure/azure-cosmos-python#141]
Similar to [azure-cosmos-js's issue #184](https://github.com/Azure/azure-cosmos-js/issues/184), the Python SDK lacks the feature parity to the [.NET CosmosClient class](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.cosmosclient?view=azure-dotnet-preview). A bit of a bummer; in the meantime, will have to parse a provided connection string. If the SDK is being provided in another language, having the same features across the original and other SDKs should be expected.

**Comment by wkeithvan** [comment]
+1... Azure needs to be able to support parsing its own format and not rely on the end user to write a parsing script for this.

**Comment by MaLiN2223** [comment]
Looks like this is solved in version 4.0.0b2
[from the changelog](https://pypi.org/project/azure-cosmos/):
> A new classmethod constructor has been added to CosmosClient to enable creation via a connection string retrieved from the Azure portal.

You can now 

```
from azure.cosmos import cosmos_client
cosmos_client.CosmosClient.from_connection_string(connection_string)
```
